pkgs/by-name/ow/ownserver/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
```

nit: I think finalAttrs might be preferable to `rec` because for this builder, it allows the individual package to be overridden with `overrideAttrs`

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

Done.
I also removed optional `apple_sdk` for darwin because I saw that [darwin builds failed because of it](https://github.com/NixOS/nixpkgs/actions/runs/20045721001/job/57490732783?pr=469114).
I have no idea about darwin so tell me if I did the right thing.